docs/application/native/tutorials/details/tizen-apis.md

```diff
@@ -38,7 +38,7 @@ APIs to use.
 > **Note**  
 > Except as noted, the mobile native APIs are available since Tizen 2.3.
 
-** Table: Mobile natvie API modules **
+**Table: Mobile natvie API modules**
```

Comment:
native is correct spelling.

Reply:
When LB reviewed, they gave a comment. We had better use "native" instead of "Native".
Please see TD.

Reply:
All contents were moved to native/guides/index.md so I delete it.